### PR TITLE
Fix permissions instructions

### PR DESCRIPTION
**Pull request by maresb**
This fixes the issue pointed out in https://github.com/labelle-org/labelle/issues/4#issuecomment-2083375488

### CODE REVIEW

**Review by tomers — APPROVED** [review]
Thanks for this!

**Comment by tomers** [comment]
@maresb, I tried creating a new release, and failed: https://github.com/labelle-org/labelle/actions/runs/8914282659
I've also added a PR for capturing instructions on how to release, based on your comment in some other PR: https://github.com/labelle-org/labelle/pull/41

Please advice how to resolve the relase issue.

**Comment by maresb** [comment]
Hi @tomers, I think I see the issue. You tagged the release as `1.2.2`, but the branch protection rules expect `v1.2.2`.

image

That corresponds to the error message:

image

You could try editing the release and changing the tag to `v1.2.2`, but I'm not sure that will trigger a new CI run. If you delete the release and delete the tag, you should be able to publish a new release.

Just in case you run into trouble with "Generate release notes" after deleting the current release, I've copied it below for safety:

---

This is mainly a maintenance relase, with no new-features.

## What's Changed

### Maintenance
* Add some type hints by @maresb in https://github.com/labelle-org/labelle/pull/22
* Don't assume the default barcode type when handling empty string by @maresb in https://github.com/labelle-org/labelle/pull/26
* Add a test to ensure that the GUI module is importable by @maresb in https://github.com/labelle-org/labelle/pull/29
* Revert unintentional interface change by @maresb in https://github.com/labelle-org/labelle/pull/30
* Fix --output=console-inverted by @maresb in https://github.com/labelle-org/labelle/pull/33
* Fix descriptions of some CLI args by @maresb in https://github.com/labelle-org/labelle/pull/34
* Add deprecation errors for old dymoprint arguments by @maresb in https://github.com/labelle-org/labelle/pull/35
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/labelle-org/labelle/pull/36
* Fix permissions instructions by @maresb in https://github.com/labelle-org/labelle/pull/38


**Full Changelog**: https://github.com/labelle-org/labelle/compare/v1.2.1...1.2.2

File: src/labelle/lib/devices/usb_device.py [hunk 1]
```diff
@@ -165,26 +164,28 @@ def _instruct_on_access_denied_linux(self) -> NoReturn:
...
-            echo '{udev_rule}' | sudo tee /etc/udev/rules.d/91-labelle-{self._dev.idProduct:x}.rules
+                echo '{udev_rule}' \\
+                  | sudo tee /etc/udev/rules.d/91-labelle-{self.id_vendor:04x}-{self.id_product:04x}.rules
 
             Next, refresh udev with:
 
-            sudo udevadm control --reload-rules
-            sudo udevadm trigger --attr-match=idVendor="0922"
+                sudo udevadm control --reload-rules
+                sudo udevadm trigger --attr-match=idVendor="0922"
```
Comment (melroy89) [inline src/labelle/lib/devices/usb_device.py:188]: Is `sudo udevadm trigger` not sufficient?

**Pull request merged**